.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,14 @@
-Fixes # .
+Fixes #
 
 #### Changes proposed in this Pull Request:
 
 
 #### Testing instructions:
--
+
+*
+
+#### Proposed changelog entry for your changes:
```

Comment:
Perhaps we can change this to:
```
<!-- Add the following only if this is meant to be in changelog --!>
#### Proposed changelog entry:
```
don't want everyone to feel like that they _have_ to write it. Otherwise it will be harder to separate the wheat from the chaff later.